requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
when creating an rpm we can't tie to specific versions because we can't exactly dictate what versions of each dependency will exist in the OS distribution. This means when creating an RPM we'll have to have a requirements.txt that looks something like this.

Now we do have options here though. One option was to try to get upstream (here on github) to implement this change upstream in the code. The other option is to patch the requirements.txt file during RPM creation. We were attempting to go with the first option, but it's not required as we can patch this in the RPM if you would prefer that.